services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None
+) -> QuerySet | MovieSession:
+    movies_session = MovieSession.objects.all()
+
+    if session_date:
+        movies_session = movies_session.filter(show_time__date=session_date)
+
+    return movies_session
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
Using `get()` can raise a `DoesNotExist` exception if no `MovieSession` with the given `id` exists. Consider handling the exception appropriately to avoid runtime errors.